backend/src/main/resources/sql/create_regions.sql

```diff
@@ -1,92 +0,0 @@
-INSERT INTO Regions ("prefix", "name", "regionIdentifier", "website") VALUES ('Landkreis', 'Aichach-Friedberg', '09771', 'https://lra-aic-fdb.de/hier-leben/ehrenamt/bayerische-ehrenamtskarte') ON CONFLICT ("regionIdentifier") DO UPDATE SET "prefix"='Landkreis', "name"='Aichach-Friedberg', "website"='https://lra-aic-fdb.de/hier-leben/ehrenamt/bayerische-ehrenamtskarte';
```

Comment:
There was documentation on how to create this sql file. We should probably adjust that documentation.

Reply:
Will do in #540